### PR TITLE
Fix lint manager

### PR DESCRIPTION
**Pull request by liorblob**

### CODE REVIEW

File: demisto_sdk/commands/lint/lint_manager.py [hunk 1]
```diff
@@ -154,7 +154,7 @@ def run_parallel_packages(self, pkgs_to_run: List[str]) -> int:
         # when we modify the file for mypy includes
         if 'Scripts/CommonServerPython' in pkgs_to_run:
             pkgs_to_run.remove('Scripts/CommonServerPython')
-            common_server_path = get_common_server_path(self.configuration.env_dir)
+            common_server_path = get_common_server_dir('')
```
Comment (yuvalbenshalom) [inline demisto_sdk/commands/lint/lint_manager.py]: why did you remove the `env_dir`?

Comment (liorblob) [reply demisto_sdk/commands/lint/lint_manager.py]: so it will be like the previous version

Comment (liorblob) [reply demisto_sdk/commands/lint/lint_manager.py]: previously it was just `Scripts/CommonServerPython`

**Comment by bakatzir** [comment]
i'm restoring the branch to be able to test the content build with it
